### ISSUE TITLE
Add checker support

### ISSUE THREAD

**Issue by vstrimaitis** [xalanq/cf-tool#105]
There are a lot of problems where multiple answers are accepted. In such a case, the `cf` tool returns "Wrong Answer" unless the output matches `ansK.txt` exactly.

It would be nice to have an option to write a checker and instruct `cf` to use it when running tests. One option I can think of is to check for a `checker.*` file in the problem's directory and use it if it exists.

Some additional features that would come in handy:
* A few predefined checkers for common use cases (like in polygon)
* Ability to generate a new custom checker for a problem (e.g. by running `cf gen checker` or something similar)
* Ability to use templates when generating custom checkers (to fill in all boilerplate for you)

**Comment by cubercsl** [comment]
Is it similar with #72 ?

**Comment by vstrimaitis** [comment]
> Is it similar with #72 ?

Yep, that's a similar idea, but that issue is only concerned with case sensitivity and precision checks. What I had in mind is to allow to optionally use completely custom checkers (i.e. when a problem has multiple possible outputs). Anyway, if this would become overcomplicated, then the case and precisions checks are a great step forward and would cover many cases 👍
